site/content/en/docs/formats/mars.md

```diff
@@ -0,0 +1,76 @@
+---
+title: 'MARS'
+linkTitle: 'MARS'
+description: ''
+weight: 14
+---
+
+## Format specification
+
+MARS is a dataset for and motion analysis and person identification task,
+and this dataset it's extension of Market-1501 dataset format.
+MARS dataset is available for downloading
+[here](http://zheng-lab.cecs.anu.edu.au/Project/project_mars.html)
+
+Supported types of annotations:
+- `Label`
+
+Required attributes:
+- `is_distractors` (bool): True when image with distractors,
+    which negatively affect retrieval accuracy
+- `is_junk`: True for junk image which do not affect retrieval accuracy;
+- `pedestrian_id`: four-digit number in format `%04d`;
```

Comment:
I'm confused by this. First of all, isn't it redundant to say `%04d` when you already said it's a four-digit number? Second, what data type is this attribute supposed to have? If `int`, then the format doesn't make sense. If `str`, then why not `int`?

In addition, the meaning of the attribute should be explained beyond just "number".